fau-degree-program-output.php

```diff
@@ -4,7 +4,7 @@
  * Plugin Name: FAU Degree Program Output
  * Plugin URI:  https://github.com/RRZE-Webteam/FAU-Studium-Embed
  * Description: Fetch degree programs via the REST API and display them.
- * Version:     2.2.1
+ * Version:     2.2.2-beta
```

Comment:
First time I see such a version update. Is this a new convention or something?

Reply:
I believe we consistently use this versioning system on FAU.

Reply:
🤯 never mind me then

Reply:
Actually, we use versioned beta releases.
```suggestion
 * Version:     2.2.2-beta.1
```